packages/web/src/SplunkErrorInstrumentation.ts

```diff
@@ -217,6 +222,24 @@ export class SplunkErrorInstrumentation extends InstrumentationBase {
 		this.endSpanWithThrottle(span, now)
 	}
 
+	private attachClearingInterval(): void {
+		this.clearingIntervalId = setInterval(() => {
+			const callback = () => {
+				this.throttleMap.forEach((relativeTime, throttleKey) => {
+					if (performance.now() - relativeTime > THROTTLE_LIMIT) {
+						this.throttleMap.delete(throttleKey)
```

Comment:
Just curious, can a race condition happen here if this is running on interval and sth else is trying to add to the map. Should it be considered ?

Reply:
JavaScript is single-threaded, and this operation is synchronous, so there is no way it would get interrupted by the add operation.